changelog/april2024/2024-04-03-block-storage-added-low-latency-offers-integration-in-c.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Low Latency offers integration in Cockpit
+status: added
+author:
+    fullname: 'Join the #block-storage channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-04-03
+category: storage
+product: block-storage
+---
+
+A new dashboard dedicated to block storage Low Latency offers is now available.
```

Comment:
```suggestion
A new dashboard dedicated to Block Storage Low Latency offers is now available.
```

Not sure, but either all uppercase or all lowercase I'd say

Reply:
I've edited your comments Bene!